srcpkgs/occt/template

```diff
@@ -11,13 +11,16 @@ makedepends="freetype-devel glu-devel freeimage-devel gl2ps-devel tbb-devel
  tcl-devel tk-devel"
 short_desc="OpenCASCADE Technology - library for CAD/CAM/CAE applications"
 maintainer="Piraty <piraty1@inbox.ru>"
-license="LGPL-2.1"
+license="LGPL-2.1-only,OpenCascade-1.0-LGPL-exception-not-SPDX"
```

Comment:
missing space between comma and start of another license.

wtf is that second thing

Reply:
rechecked and fixed